frontend/public/style/_vars.scss

```
@@ -14,6 +14,12 @@ $consoleWebDistPath: './';
 // PatternFly font path.
 $pf-v6-global--font-path: '~@patternfly/patternfly/assets/fonts';
 
+// PatternFly icon font path.
+$pf-v6-global--fonticon-path: '~@patternfly/patternfly/assets/pficon';
```

Stylelint (17.5.0)

[error] Expected variable name "pf-v6-global--fonticon-path" to be kebab-case (scss/dollar-variable-pattern)

(scss/dollar-variable-pattern)